include/rcutils/types/array_list.h

```diff
@@ -0,0 +1,208 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
@nburek `2017` -> `2019` ?

Reply:
Corrected all of these to be `2018-2019`, since the work started in 2018 and is being continued now in 2019.